wolfcrypt/src/curve25519.c

```diff
@@ -251,6 +255,10 @@ int wc_curve25519_shared_secret_ex(curve25519_key* private_key,
                                    curve25519_key* public_key,
                                    byte* out, word32* outlen, int endian)
 {
+#ifdef WOLFSSL_SE050
+    (void)endian;
+    int ret = se050_curve25519_shared_secret(private_key, public_key, out, outlen);
```

Comment:
Please have `se050_curve25519_shared_secret` accept the endian argument and mark unused inside it.

Reply:
I would prefer the `int ret` be defined above the #ifdef in all cases. Cleaner, since the `return ret` always exists.